library/x509_invasive.h

```diff
@@ -0,0 +1,20 @@
+
+#ifndef MBEDTLS_X509_INVASIVE_H
```

Comment:
The CI reminds me that when adding a new file, one needs to then run `tests/scripts/check-generated-files.sh -u` and commit the modified files.

Reply:
:+1: